mlflow/server/js/src/components/ExperimentView.js

```diff
@@ -88,6 +88,9 @@ export class ExperimentView extends Component {
     // Input to the lifecycleFilter field
     lifecycleFilter: PropTypes.string.isRequired,
 
+    orderByKey: PropTypes.string,
+    orderByAsc: PropTypes.bool.isRequired,
```

Comment:
```suggestion
    orderByAsc: PropTypes.bool,
```

Reply:
orderByKey is nullable, but orderByAsc is not